docs/pages/introduction.mdx

```diff
@@ -1,13 +1,21 @@
-# MUD Introduction
+# What is MUD?
 
-MUD is a framework for ambitious Ethereum applications. It compresses the complexity of building EVM apps with a tightly integrated software stack.
+MUD is a framework for ambitious onchain applications. It compresses the complexity of building [Ethereum](https://ethereum.org/) apps with a tightly integrated software stack. It's [open source](https://github.com/latticexyz/mud) and [free to use](https://github.com/latticexyz/mud/blob/main/LICENSE).
 
-MUD evolved out of our experiences building onchain games like [Dark Forest](https://zkga.me/) and zkDungeon. We identified many difficulties along the way: coupling of state and logic in smart contracts makes upgrading logic difficult, lack of synchrony between the chain and client can lead to inconsistencies in game state, irregularities with access controls pose problems for would-be third-party developers attempting to create their own plugins and clients. This revealed the need for a framework and a protocol to handle the inevitable complexity of game code, and to counteract the developer-unfriendly patterns inherent in the way smart contracts are traditionally written. We built MUD with solutions to these problems in mind.
+## Why MUD?
```

Comment:
For me, this is still a lot to read, especially being the first actual content after the mud.dev home page. 

I wonder if we can make "Why MUD" even more succinct (perhaps bullet points), and merge it with the (tiny) Quick Start page. Then we can move the long form Introduction to a separate page. 

Similar to https://viem.sh/docs/getting-started

Reply:
any suggestions on how you'd distill it into bullets?

I thought painting a picture of before/after helps build some context, since it's not only about the features we offer but also how big a difference it makes from traditional onchain apps

Reply:
landing this, feel free to offer suggestions on how to continue to improve this page